kickstarts/base.ks.erb

```diff
@@ -111,7 +108,7 @@ echo "" >> /etc/ssh/sshd_config
 dnf -C -y --noplugins remove linux-firmware doxygen
 
 # Ensure packages are up to date
-dnf -y update --nobest
```

Comment:
We shouldn't need the `--nobest`.  I think it was added for an ansible rpm issue, but we're on a much newer version now and there shouldn't be any dependency issues anymore.  (I didn't see any issues during install or update of my test appliance)